### PR TITLE
Bring the French file in the official order of Crowdin.

### PR DESCRIPTION
**Pull request by teolemon**
Bring the French file in the official order of Crowdin. Please don't force edit the French po in the future.

### CODE REVIEW

File: po/common/fr.po [hunk 1]
```diff
@@ -2376,6 +2374,10 @@ msgctxt "sort_by"
 msgid "Sort by"
 msgstr "Trier par"
 
+msgctxt "sort_completeness"
+msgid "Completeness"
+msgstr "Entièreté"
```
Comment (stephanegigandet) [inline po/common/fr.po]: Complétude

**Comment by stephanegigandet** [comment]
"Bring the French file in the official order of Crowdin. Please don't force edit the French po in the future."

@teolemon : what does that mean?

Usually when I add entries to the po file, I put them in common.pot, en.po and fr.po. Can I do that?

File: po/common/fr.po (hunk 1 above)
Comment (teolemon) [inline po/common/fr.po]:
```suggestion
msgstr "Complétude"
```

**Comment by VaiTon** [comment]
> 
> 
> "Bring the French file in the official order of Crowdin. Please don't force edit the French po in the future."
> 
> @teolemon : what does that mean?
> 
> Usually when I add entries to the po file, I put them in common.pot, en.po and fr.po. Can I do that?

I think it's better to add them just to the common file. 

I, too, used to add them to en.po but I will fix my bad habit :-)

**Pull request merged**